faculty/clients/experiment.py

```diff
@@ -293,3 +331,30 @@ def list_runs(
         return self._get(
             endpoint, ListExperimentRunsResponseSchema(), params=query_params
         )
+
+    def log_run_data(
+        self, project_id, run_id, metrics=None, params=None, tags=None
+    ):
+        """Update the data of a run.
+
+        Parameters
+        ----------
+        project_id : uuid.UUID
+        run_id : uuid.UUID
+        metrics : List[Metric], optional
+            Each metric will be inserted.
+        params : List[Metric], optional
+            Each param will be inserted and rejected if the given key is
+            present.
+        tags : List[Tag], optional
+            Each tag be upserted.
```

Comment:
Should we add a note to say -- if any params conflict, the entire operation is rejected?

Reply:
👍